nautobot/docs/models/extras/relationship.md

```diff
@@ -9,11 +9,40 @@ Sometimes it is desirable to create a new kind of relationship between one (or m
 
 ## Relationship Filters
 
-Filters can be defined to restrict the type or selection of objects for either side of the connection. From the FEX example above, you can restrict the FEX side of the connection to only of devices with the FEX role, and restrict the controller side to only devices with Controller roles.
+Filters can be defined to restrict the type or selection of objects for either side of the connection. 
+
+### Relationship Filter Example
```

Comment:
I don't know that this subheading is really necessary here, as this is the only subsection under the parent section heading.